README.md

```diff
@@ -1,8 +1,11 @@
 # JupyterLite
 
-[![ci-badge]][ci] [![binder-badge]][binder] [![docs-badge]][docs]
+[![ci-badge]][ci] [![lite-badge]][lite] [![binder-badge]][binder] [![docs-badge]][docs]
 
 [ci-badge]: https://github.com/jupyterlite/jupyterlite/workflows/Build/badge.svg
+[lite-badge]:
+  https://gist.githubusercontent.com/bollwyvl/72565d11c9f0340c98c8199438cd63d5/raw/d78510cafbc45f83eba2acf51a38e5885d0bb3ce/badge.svg
```

Comment:
ha, i forgot we were using that gist. lets _at least_ get it into this repo, i suppose...

Reply:
https://github.com/jupyterlite/jupyterlite/pull/491